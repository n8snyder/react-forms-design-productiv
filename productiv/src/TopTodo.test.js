import TopTodo from './TopTodo';
import { defaultTodos } from './App';
import { render } from "@testing-library/react";

describe("Todo", function () {
  it("renders without crashing", function () {
    render(<TopTodo todos={defaultTodos}/>);
  });

  it("snapshot", function () {
    const { container } = render(<TopTodo todos={defaultTodos}/>);
    expect(container).toMatchSnapshot();
  });

  it("presentation of Todo", function(){
    const { container } = render(<TopTodo todos={defaultTodos}/>);
    expect(container.querySelector(".Todo")).toBeInTheDocument();
    expect(container.querySelector(".Todo")).toContainHTML("priority: 1");
    expect(container.querySelector(".Todo")).not.toContainHTML("priority: 4");
  });
});