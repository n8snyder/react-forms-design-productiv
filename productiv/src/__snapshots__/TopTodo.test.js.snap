// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo snapshot 1`] = `
<div>
  <div
    class="TopTodo"
  >
    <div
      class="Todo"
    >
      <div>
        <b>
          Make dinner
        </b>
         
        <small>
          (priority: 
          1
          )
        </small>
      </div>
      <div>
        <small>
          Cook something healthy
        </small>
      </div>
    </div>
  </div>
</div>
`;
